fluent/src/parser.js

```diff
@@ -72,7 +74,8 @@ class RuntimeParser {
     const ch = this._source[this._index];
 
     // We don't care about comments or sections at runtime
-    if (ch === '/') {
+    if (ch === '/' ||
+      (ch === '#' && [' ', '#'].includes(this._source[this._index + 1]))) {
```

Comment:
You may need to skip also if `ch === '#' && ch2 === '\n'`.

Reply:
This has to be fixed also on master in #149. I'll backport it together with other changes when master becomes 0.6.1.